app/controllers/registrations_controller.rb

```diff
@@ -16,7 +16,7 @@ def create
     super do |resource|
       if resource.persisted?
         resource.add_role(:adopter, Current.organization)
-        FormSubmission.create!(person_id: resource.person_id)
+        resource.person.form_submissions.create
```

Comment:
Same here https://github.com/rubyforgood/pet-rescue/pull/997/commits/d948471b7f2cafe885a10c0523b8932bb0e40dab#r1781695089

Reply:
We can't use that with has_many though.

Reply:
Oh really? Ok I will have to look into that tomorrow when I am at my computer. I thought we could use that helper but I may be wrong.

Reply:
Ah yes, I tested this in the console and you're right.